package.json

```diff
@@ -7,23 +7,71 @@
   "icon": "icon.png",
   "license": "Apache-2.0",
   "engines": {
-    "podman-desktop": "^0.16.0"
+    "podman-desktop": ">=1.8.0"
   },
   "main": "./dist/extension.cjs",
   "source": "./src/extension.ts",
   "contributes": {
     "configuration": {
       "title": "Red Hat OpenShift Local",
       "properties": {
-        "OpenShift-Local.memory": {
+        "crc.factory.openshift.memory": {
           "type": "number",
           "format": "memory",
-          "description": "Memory size, in MiB"
+          "minimum": 11000000000,
```

Comment:
I don't understand this default as I have 5GB when displaying the wizard. And CRC does not accept lower that 496Mib

Reply:
Because there is no way to customize the minimum/maximum of a setting at runtime i had to create 2 settings (one for microshift, one for openshift local). This one is related to openshift. 
The one about microshift is here https://github.com/crc-org/crc-extension/pull/228/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R36
Based on the preset selected by the user when setting it up, it picks and shows the correct setting